### PR TITLE
Testing view ensemble_overview

### PR DESCRIPTION
**Pull request by oysteoh**
Relates to #36

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
-from unittest.mock import Mock, MagicMock
+
+from tests.data.snake_oil_data import ensembles_response
+from selenium.webdriver.chrome.options import Options
+
+
+def pytest_setup_options():
+    options = Options()
+    options.add_argument("--headless")
+    options.add_argument("--disable-gpu")
```
Comment (ManInFez) [inline tests/conftest.py:11]: Some of our components are hardware accelerated, will disable-gpu affect testing those?

Comment (oysteoh) [reply tests/conftest.py:11]: hmm... i guess we can remove that and only run with `--headless` ? I think that should be sufficient! I'll change and check if the tests are running 👍

**Review by ManInFez — APPROVED** [review]
Good first step in testing dash! We should strive to test all our views :)

**Comment by oysteoh** [comment]
> Good first step in testing dash! We should strive to test all our views :)

Yes, i suggest we are not closing the issue before all views are tested! 👍

**Pull request merged**